lab_4_fill_words_by_ngrams/main.py

```diff
@@ -98,6 +143,42 @@ def run(self, seq_len: int, prompt: str) -> str:  # type: ignore
                 or if sequence has inappropriate length,
                 or if methods used return None.
         """
+        if not (isinstance(prompt, str) and prompt
+                and isinstance(seq_len, int) and seq_len > 0
+        ):
+            raise ValueError
+
+        encoded = self._word_processor.encode(prompt)
+
+        if not encoded:
+            raise ValueError
+
+        while seq_len >= 1:
```

Comment:
Is there any variation in terms of how many steps there will be? If not, `for` should be used